### PR TITLE
ui-enhancement #1340 - conversion of normal toast to Material Design …

### LINKED ISSUE #1340: Conversion of Normal Toast to Material Design Toast -UI Enhancement
**Summary:** 

Normal toast are required to display any short message in the app but it doesn't tells the user it's a warning or info or error and doesn't look quite interesting , so converting normal toast to Material Design toast would be better option for wherever normal toast is there .
I have added toast ->  MD toast for dashboard activity which have TYPE_INFO toast with symbol.

**Before MD Toast ** 

before md

**After MD Toast :** 

It will have well defined Toast Message which looks great with certain icons of Info , Warning , Success etc... 

**Device and Android version:** 
I tested it on Physical Samsung J7000 and Realme X phone and various emulators as well , it works fine. It is also used in many production level apps too.

**Various types of Toast Message with Icon :** 
success

info

warning

error

### PR DESCRIPTION
**Pull request by rohanprasadofficial**
This converts the normal toast message in the Login Activity and Dashboard Activity to Material Design which looks better than normal toast with icon of warnings , info , success etc.. 

Fixes #1340 

Screenshots 

**Login**

> Before 

beforelogin

> After

loginafter

**Dashboard**

> Before

beforedashboard

> After

afterdb

Please make sure these boxes are checked before submitting your pull request - thanks!

- [x] Apply the `MifosStyle.xml` style template to your code in Android Studio.

- [x] Run the unit tests with `./gradlew check` to make sure you didn't break anything

- [x] If you have multiple commits please combine them into one commit by squashing them.

### CODE REVIEW

File: mifosng-android/src/main/java/com/mifos/mifosxdroid/login/LoginActivity.java [hunk 1]
```diff
@@ -186,8 +189,11 @@ public void onLoginSuccessful(User user) {
         // Saving user
         PrefManager.saveUser(user);
 
-        Toast.makeText(this, getString(R.string.toast_welcome) + " " + user.getUsername(),
-                Toast.LENGTH_SHORT).show();
+
+        FancyToast.makeText(this, getString(R.string.toast_welcome) + " "
+                + user.getUsername(), FancyToast.LENGTH_SHORT, FancyToast.SUCCESS,false).show();
```
Comment (robustTechie) [inline mifosng-android/src/main/java/com/mifos/mifosxdroid/login/LoginActivity.java]: Here, after SUCCESS ',' is not followed by whitespace

**Comment by RajaVamsi11** [comment]
@rohanprasadofficial Can you please resolve the Travis CI errors. You can locally run the command `./gradlew check` to know the reason for the Travis fail.

**Comment by rohanprasadofficial** [comment]
@RajaVamsi11  hello , i am resolving it locally , soon it will be resolved , thanks

**Comment by RajaVamsi11** [comment]
@rohanprasadofficial Also please squash your commits into one to maintain good commit history. It is not recommended to maintain more than one commit per PR.

**Comment by AbhilashG97** [comment]
Hello @rohanprasadofficial, 

The changes made in this pull request don't seem to match with the solution that you wanted to implement.

I'm closing your pull request for now. Please re-open this pull request after you have implemented your solution correctly. 

Thank you for your contribution.

**Pull request closed without merging**